### ISSUE TITLE
[QUESTION] CQL evaluation is  extremely low in minari/walker2d-v0

### ISSUE THREAD

**Issue by zugbl** [takuseno/d3rlpy#466]
Run the following code:

    dataset, env = d3rlpy.datasets.get_minari("mujoco/walker2d/medium-v0")

    # fix seed
    d3rlpy.seed(args.seed)
    d3rlpy.envs.seed_env(env, args.seed)

    encoder = d3rlpy.models.encoders.VectorEncoderFactory([256, 256, 256])

    
    conservative_weight = 10.0
    
      #  conservative_weight = 5.0

    cql = d3rlpy.algos.CQLConfig(
        actor_learning_rate=1e-4,
        critic_learning_rate=3e-4,
        temp_learning_rate=1e-4,
        actor_encoder_factory=encoder,
        critic_encoder_factory=encoder,
        batch_size=256,
        n_action_samples=10,
        alpha_learning_rate=0.0,
        conservative_weight=conservative_weight,
        compile_graph=args.compile,
    ).create(device=args.gpu)

    cql.fit(
        dataset,
        n_steps=500000,
        n_steps_per_epoch=1000,
        save_interval=10,
        evaluators={"environment": d3rlpy.metrics.EnvironmentEvaluator(env)},
        experiment_name=f"CQL_{args.dataset}_{args.seed}",
    )

At about 93k step, the environment always be:'environment': -13.130258737612,'environment': -13.197296292005936,'environment': -12.090683303073908
And it keeps fluctuate among -13,-12,-11.
Thank you for your work on this excellent project!
